src/decorators/Mention/Suggestion/index.js

```diff
@@ -246,18 +254,24 @@ function getSuggestionComponent() {
           showSuggestions: result.length > 0,
           showLoader: false,
         });
+        if (result.length === 0) {
+          this.closeSuggestionDropdown();
+        }
       });
     });
 
-    updateSuggestions = (props) => {
-      const mentionText = props.children[0].props.text.substr(1);
-
+    updateSuggestions = () => {
+      const mentionText = this.props.children[0].props.text.substr(1);
       if (config.fetchSuggestions) {
-        this.setState({
-          showLoader: true,
-        });
-        this.debouncedFetchSuggestion(mentionText);
+        if (mentionText === '' || this.state.showSuggestions) {
+          this.openSuggestionDropdown();
+          this.setState({
+            showLoader: true,
+          });
+          this.debouncedFetchSuggestion(mentionText);
+        }
       } else {
+        this.openSuggestionDropdown();
```

Comment:
Please check the appearance of mention component in case of mentionText with space (for e.g. First Name) and type space and then delete the space.